client/response.go

```diff
@@ -56,9 +56,9 @@ func (resp *Response) Update() (data []byte, err error) {
 }
 
 func (resp *Response) Status() (status *Status, err error) {
-	data := bytes.SplitN(resp.Data, []byte{'\x00'}, 2)
-	if len(data) != 2 {
-		err = fmt.Errorf("Invalid data: %v", resp.Data)
+	data := bytes.Split(resp.Data, []byte{'\x00'})
```

Comment:
This seems like a regression. the data after the first `'\x00'` could have null bytes, and this change would remove them from data[1] which gets called below.

Reply:
@jacksonrnewhouse , or interesting, the previous version SplitN was failing with the example client's requests because they contained I believe 3 elements after split, which one is right?

Reply:
reverted to original version, appears to have been a fluke on my side.